test/api/client/CentralServerService.ts

```diff
@@ -8,6 +8,7 @@ import BaseApi from './utils/BaseApi';
 import BillingApi from './BillingApi';
```

Comment:
Do not touch this file, it's deprecated.

Reply:
Did i have to delete this line ?

Reply:
Yes, remove your changes